rimage/image_file.go

```diff
@@ -180,7 +180,11 @@ func WriteImageToFile(path string, img image.Image) (err error) {
 // ConvertImage converts a go image into our Image type.
 func ConvertImage(img image.Image) *Image {
```

Comment:
could we provide a `ConvertImageSafe` to at least give end users a safe option?

Reply:
Added